### PR TITLE
Add dnslib recipe

### PR DESCRIPTION
**Pull request by mwilson8**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/anyio/meta.yaml [hunk 1]
```diff
@@ -1,41 +0,0 @@
-{% set name = "anyio" %}
```
Comment (scopatz) [inline recipes/anyio/meta.yaml]: Please do not remove other recipes

Comment (mwilson8) [reply recipes/anyio/meta.yaml]: Even ones that were removed from rebasing on the master?

**Comment by mwilson8** [comment]
@oblute @benhuff will you be maintainers?

**Comment by benhuff** [comment]
yes 👍

**Comment by oblute** [comment]
yes!

**Comment by mwilson8** [comment]
@conda-forge/staged-recipes this package is ready for final review & merge

**Comment by mwilson8** [comment]
@marcelotrevisani this is ready for final review & merge

File: recipes/dnslib/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  run:
+    - python
+
+test:
+  imports:
+    - dnslib
+
+about:
+  home: "http://bitbucket.org/paulc/dnslib/"
+  license: "BSD"
```
Comment (scopatz) [inline recipes/dnslib/meta.yaml]: this should be bsd- 2-clause

Comment (mwilson8) [reply recipes/dnslib/meta.yaml]: @scopatz fixed & test re-built; ready for review

**Comment by mwilson8** [comment]
@conda-forge/staged-recipes this package is ready for review

**Pull request merged**